.github/workflows/pr_pre-commit.yaml

```
@@ -14,6 +17,22 @@ jobs:
       - uses: actions/setup-python@82c7e631bb3cdc910f68e0081d67478d79c6982d # v5.1.0
         with:
           python-version: '3.12'
+      - name: Setup Hadolint
+        shell: bash
+        run: |
+          set -xeuo pipefail
```

We can remove debug now, if you want. I don't mind if it stays in though.

```suggestion
          set -euo pipefail
```